app/models/event_type.rb

```diff
@@ -1,15 +1,15 @@
 class EventType
   ALL =
     {
-      "Train to Teach event" => 222_750_001,
+      "Get Into Teaching event" => 222_750_001,
```

Comment:
@EllieNodder It is very likely that we will have a new event type so this shouldn't be renamed (along with some of the other event functionality stuff - better to leave to Ross!)